paywall/src/data-iframe/blockchainHandler/account.js

```diff
@@ -26,10 +25,8 @@ export async function pollForAccountChange(
   web3Service,
   onAccountChange = () => {}
 ) {
-  await ensureWalletReady(walletService)
```

Comment:
this removal enables account polling even when we have no account (and is a critical change)